.github/workflows/check-tutorial.yaml

```diff
@@ -0,0 +1,72 @@
+name: "Check tutorial"
+
+on:
+  push:
+    branches:
+    - master
+    - release
+  pull_request:
+  schedule:
+    # Everyday at 4:00 AM
+    - cron: "0 4 * * *"
+
+jobs:
+  check-tutorial:
+    name: "Check tutorial"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Check links
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        folder-path: "docs/docs/tutorial"
+        config-file: ".github/workflows/check-links-config.json"
+
+    # TODO: use javascript? https://github.com/marketplace/actions/execute-javascript-inline
+
+    - name: Using currently supported cardano-node version?
+      shell: python
+      run: |
+        import re
```

Comment:
Suggestion: do not write inline scripts but call scripts from `.github/scripts` repository. This will make it easier to test those scripts as they can be easily run locally

Reply:
Seems like this is harder than thought. The `runs:` will make the script be saved in a file and run by `python <file>` and I couldn't make python import from a specific path easily. I'll deem this as not worth it and just keep the scripts inlined for now.